index.html

```diff
@@ -91,124 +92,44 @@ <h2>Multi-Post Stories</h2>
-                </div>
-
-                <div class="showcase-detail">
-                  <h2>Profesional Art Printing Data</h2>
-                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
-                  <ul>
-                      <li>HTML</li>
-                      <li>Bootstrap</li>
-                      <li>Ruby</li>
-                  </ul>
-
-              </div>
-              <button id="showcase-button"  class="showcase-btn">See project</button>
-            </div>
 
-            <div class="showcase" id="showcase">
-                <div class="showcase-image">
-                   <img src="images/Img Placeholder (3).png" alt="project image">
-                </div>
+        <div class="project-container" id="project-container">
 
-                <div class="showcase-detail">
-                  <h2>Profesional Art Printing Data</h2>
-                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
-                  <ul>
-                      <li>HTML</li>
-                      <li>Bootstrap</li>
-                      <li>Ruby</li>
-                  </ul>
-
-              </div>
-              <button id="showcase-button"  class="showcase-btn" type="button">See project</button>
-            </div>
+        </div>
 
-            <div class="showcase" id="showcase">
-                <div class="showcase-image">
-                   <img src="images/Img Placeholder (2).png" alt="project image">
-                </div>
 
-                <div class="showcase-detail">
-                  <h2>Profesional Art Printing Data</h2>
-                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
-                  <ul>
-                      <li>HTML</li>
-                      <li>Bootstrap</li>
-                      <li>Ruby</li>
-                  </ul>
-              </div>
-              <button id="showcase-button"  class="showcase-btn" type="button">See project</button>
-            </div>
 
-            <div class="showcase" id="showcase">
-                <div class="showcase-image">
-                    <img src="images/Img Placeholder (4).png" alt="project image">
-                </div>
-                <div class="showcase-detail">
-                  <h2>Profesional Art Printing Data</h2>
-                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
-                  <ul>
-                      <li>HTML</li>
-                      <li>Bootstrap</li>
-                      <li>Ruby</li>
-                  </ul>
-              </div>
-              <button id="showcase-button"  class="showcase-btn" type="button">See project</button>
+    <div id="popup" class="popup">
```

Comment:
- [ ] Well done! 👏🎉 It seems you have made progress in addressing the previous reviewer's feedback and resolving the linter errors. However, there are still a few adjustments that need to be made. 

- [ ] First, as mentioned by the previous reviewer, the projects in your "Recent Works" section should be loaded dynamically using JavaScript code. This means that not only the secondary projects but also the main project should be loaded dynamically from the array. Currently, it seems that the main project is not being loaded dynamically. Please update your code to ensure that the main project is also loaded dynamically.

- [ ] Second, the pop-up content should also be populated dynamically. Based on the current state of your code, it appears that the pop-up content is not being dynamically generated based on which project the user click. Please make the necessary changes to ensure that the pop-up content is dynamically populated as required.

[Ian-Kiema-Portifolio.webm](https://github.com/iankiema/My-Portifolio-Personal/assets/55840999/bdb0c2fb-6fff-4710-8e30-d6ff13edc2b4)

Please review these points and update your code accordingly. If you have any questions or need further assistance, feel free to ask. Keep up the good work! 💪🌟